staging/src/slime.io/slime/modules/lazyload/charts/templates/cluster-global-sidecar.yaml

```diff
@@ -26,12 +26,20 @@ metadata:
     slime.io/serviceFenced: "false"
 spec:
   ports:
+    # for lazyload
```

Comment:
To reduce duplicate code, what do you think about writing it in the following way?

```go
var gsPort = default(80, gs.port)
var hasSidecarPort bool
svcPorts := make([]int, 0, len(wormholePort))
for _, p := range wormholePort {
  svcPorts = append(svcPorts, p)
  if p == gsPort {
    hasSidecarPort = true
  }
}

if !hasSidecarPort {
  svcPorts = append(svcPorts, gsPort)
}

for _, p := range svcPorts {
   // ...

   // gsPort ...

  /// ...
}

```